Choreographer_30FPS/app/src/main/jni/Common/ndk_helper/Android.mk

```diff
@@ -0,0 +1,37 @@
+LOCAL_PATH:= $(call my-dir)
+
+include $(CLEAR_VARS)
+
+LOCAL_MODULE:= ndk_helper
+LOCAL_SRC_FILES:= JNIHelper.cpp \
```

Comment:
this file need to be checked into android-mk branch if build is with ndk-build.  the main branch is for gradle build I think